### PR TITLE
Fix errors

### PR DESCRIPTION
**Pull request by christiaanwesterbeek**

### CODE REVIEW

**Comment by Cyberuben** [comment]
I still see a lot of whitespace issues. Please solve this, I'll add eslint config later on and fix the coding style to be more uniform.

**Comment by christiaanwesterbeek** [comment]
I'm not really interested to fix whitespaces. It was my editor that removed trailing spaces and added a trailing enter at the end of the files. I'll leave it up to you to take it from here.

**Comment by christiaanwesterbeek** [comment]
The whitespace issues are now solved

File: src/request.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class JSONRequest {
 								error = json.exception;
 							}
 						} catch (err) {
-
+                            reject(err)
```
Comment (Cyberuben) [inline src/request.js]: This still has whitespace issues

File: test/test.js [hunk 2]
```diff
@@ -92,12 +92,17 @@ describe("Postcode.nl API wrapper", () => {
 			});
 
 			it("should be able to start requesting the address", () => {
+                var client = new PostcodeClient({
```
Comment (Cyberuben) [inline test/test.js]: This block still has whitespace issues

**Review by Cyberuben — CHANGES_REQUESTED** [review]

**Comment by Cyberuben** [comment]
I'll fix it myself, takes too long

**Comment by Cyberuben** [comment]
https://www.npmjs.com/package/node-postcode

Published to npm

**Pull request merged**